### DISCUSSION TITLE
How to make directory in terminal same with in yazi?

### DISCUSSION THREAD

**Discussion by OriLiMu** [sxyazi/yazi#452] [1. Q&A] [answered]
For example, I start yazi in home dir, and then in yazi I enter `~/image/` folder. 
When I exit yazi, the directory in terminal is `~/image/`

**Comment by sxyazi** [comment, marked as answer]
See https://yazi-rs.github.io/docs/quick-start#changing-working-directory-when-exiting-yazi

**Reply by LinArcX** [reply]
What about cmd users who wanna use batch files? I couldn't find a `.bat` file.
